ajax/libs/imgareaselect/package.json

```diff
@@ -26,5 +26,13 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/odyniec/imgareaselect"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/odyniec/imgareaselect.git",
+    "basePath": "distfiles",
+    "files": [
+      "**!(*.txt)/*"
```

Comment:
I tried with `"**/*.!(txt)"` but didn't see any txt files been included, please confirm that.